### PR TITLE
Cybl 57 complete

### PR DESCRIPTION
**Pull request by jacobstrasburg**
Game over condition now binds properly and game ends according to selected condition.

### CODE REVIEW

File: src/models/settings-model.ts [hunk 1]
```diff
@@ -18,6 +18,11 @@ export class SettingsModel {
     schedule: Map<number, Array<number>>;
     scheduleText:string;
     selectedGameOverCondition: string = 'throwCount';
+    gameOverConditions = [
+        { id: 'throwCount', label: 'Throw Count'},
```
Comment (Drew-j-Smith) [inline src/models/settings-model.ts:22]: We may want to make this static and pass the array to Object.freeze

**Review by Drew-j-Smith — CHANGES_REQUESTED** [review]
The selectedGameOverCondition option did nothing before. Requesting changes so that all conditions work, not just the leave condition.

File: src/scenes/cyberball.ts [hunk 2]
```diff
@@ -366,10 +375,10 @@ export class CyberballScene extends Phaser.Scene {
         if (this.settings.timeLimit > 0 && this.settings.displayTimeLimit)
             this.timeLimitText.setText(this.getTimeString());
 
-        if (this.settings.timeLimit > 0 && Date.now() - this.startTime > this.settings.timeLimit) {
-            this.postEvent('global-time-limit');
-            this.gameOver();
-        }
+        // if (this.settings.timeLimit > 0 && Date.now() - this.startTime > this.settings.timeLimit) {
+        //     this.postEvent('global-time-limit');
```
Comment (Drew-j-Smith) [inline src/scenes/cyberball.ts]: Why did this get commented out?

Comment (jacobstrasburg) [reply src/scenes/cyberball.ts]: I just moved it to the beginning of the function and forgot to delete it here.

Comment (Drew-j-Smith) [reply src/scenes/cyberball.ts]: Ah, I see. If you delete this then I think this looks good to merge

**Review by Drew-j-Smith — APPROVED** [review]

**Pull request merged**